lib/diff/lcs.rb

```diff
@@ -380,14 +375,14 @@ def traverse_sequences(seq1, seq2, callbacks = Diff::LCS::SequenceCallbacks) #:y
         ai += 1
       end
 
-      next unless bj < b_size
-
-      ax = string ? seq1[ai, 1] : seq1[ai]
-      bx = string ? seq2[bj, 1] : seq2[bj]
-      event = Diff::LCS::ContextChange.new('+', ai, ax, bj, bx)
-      event = yield event if block_given?
-      callbacks.discard_b(event)
-      bj += 1
+      if bj < b_size
```

Comment:
I think writing it like this makes it more similar to the portion of code above it (which check for `ai < a_size`), which makes it easier to follow.

Reply:
I agree. I believe that I had done it the other way because Rubocop loves its guard conditions. I’m removing the Rubocop configuration (I will be switching this codebase to standardrb, but only when we are about to release 1.5 for real).